### ISSUE TITLE
Mobile needs a way to stop sphere

### ISSUE THREAD

**Issue by Jared-Sprague** [ScriptaGames/zorbio-issues#14]
So a few people have mentioned that it would be nice to slow down or stop on mobile.  I have to agree it's definitely a disadvantage on mobile vs keyboard that you can't stop.  Stopping so you can line up an intercept vector is a good strategy for capturing that I use a lot on a keyboard.  

So some way to stop on mobile is needed. like maybe a two finger touch, or a stop button that only shows up on mobile.

**Comment by mwcz** [comment]
Yeah!  I've heard this feedback too and totally agree.  Maybe pinch/spread to adjust speed (Mark's idea).  I'd rather not have a button.
